src/core_plugins/metrics/public/components/__tests__/add_delete_buttons.js

```diff
@@ -0,0 +1,67 @@
+// import React from 'react';
```

Comment:
Enzyme stopped working with our webpack configuration. I didn't want to hold up the PR but I also didn't want to loose the tests once we got things sorted out with Webpack